base_tier_validation/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,83 @@
+                                <label for="module_base_tier_validation_formula" />
+                                <div class="text-muted">
+                                    Option to use python formula to get reviewers and find documents
+                                </div>
+                            </div>
+                        </div>
+                        <!-- <div
+                            class="col-12 col-lg-6 o_setting_box"
+                            id="module_base_tier_validation_server_action"
+                        >
+                            <div class="o_setting_left_pane">
+                                <field
+                                    name="module_base_tier_validation_server_action"
+                                />
+                            </div>
+                            <div class="o_setting_right_pane">
+                                <label
+                                    for="module_base_tier_validation_server_action"
+                                />
+                                <div class="text-muted">
+                                    Option to trigger server actions on completion of tier reviews
+                                </div>
+                            </div>
+                        </div> -->
+                        <div
+                            class="col-12 col-lg-6 o_setting_box"
+                            id="module_base_tier_validation_forward"
+                        >
+                            <div class="o_setting_left_pane">
+                                <field name="module_base_tier_validation_forward" />
+                            </div>
+                            <div class="o_setting_right_pane">
+                                <label for="module_base_tier_validation_forward" />
+                                <div class="text-muted">
+                                    Option to forward a tier validation to other reviewer, and backward
+                                </div>
+                            </div>
+                        </div>
+                        <!-- <div
+                            class="col-12 col-lg-6 o_setting_box"
+                            id="module_base_tier_validation_report"
+                        >
+                            <div class="o_setting_left_pane">
+                                <field name="module_base_tier_validation_report" />
+                            </div>
+                            <div class="o_setting_right_pane">
+                                <label for="module_base_tier_validation_report" />
+                                <div class="text-muted">
+                                    More reports about tier validations, i.e., summary of all my reviews
+                                </div>
+                            </div>
+                        </div> -->
+                    </div>
+                </div>
```

Comment:
```suggestion
                                <app id="base_tier_validation">
                    <block
                        class="row mt16"
                        name="base_tier_validation_option_setting_container"
                        title="Tier Validation"
                    >
                        <setting
                            class="col-12 col-lg-6"
                            id="module_base_tier_validation_formula"
                        >
                            <div>
                                <field name="module_base_tier_validation_formula" />
                            </div>
                            <div>
                                <label for="module_base_tier_validation_formula" />
                                <div class="text-muted">
                                    Option to use python formula to get reviewers and find documents
                                </div>
                            </div>
                        </setting>
                        <setting
                            class="col-12 col-lg-6"
                            id="module_base_tier_validation_forward"
                        >
                            <div>
                                <field name="module_base_tier_validation_forward" />
                            </div>
                            <div>
                                <label for="module_base_tier_validation_forward" />
                                <div class="text-muted">
                                    Option to forward a tier validation to other reviewer, and backward
                                </div>
                            </div>
                        </setting>
                    </block>
                </app>
```

Disclaimer: I did not test this, but should work.  
This changes are proposed in the [migration guidelines](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0)

Also, I suggest deleting the commented code (check if it's no longer needed).